great-idea-website/index.html

```diff
@@ -16,48 +16,91 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="container">
+        <nav class="nav">
+            <ul style="display: flex; margin-bottom: 50px;">
+              <li>Services</li>
+              <li>Product</li>
+              <li>Vision</li>
+              <li>Features</li>
+              <li>About</li>
+              <li>Contact</li>
+              <li>
+                  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+              </li>
+            </ul>
+          </nav>
+          
+        <div class="header-text-container">
+            <h1 class="header-text"> Innovation <span class="new-line">On </span> Demand</h1>
```

Comment:
This is a clever way to break a line, but you could probably just use a `<br>` tag with the same result. 
Good job anyway to have found an original way to do so.